### PR TITLE
Improve Composer config

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
   ],
   "type": "wordpress-theme",
   "scripts": {
-    "install:dev": "rm -Rf ./composer.lock && composer install",
-    "install:prod": "rm -Rf ./composer.lock && composer install --no-dev",
+    "install:dev": "@composer update",
+    "install:prod": "@composer update --no-dev",
```
Comment (szepeviktor) [inline composer.json:13]: `composer update` ignores locked versions.
BTW `composer install` cannot do its job without the lock file.

File: composer.json [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
     "install-codestandards": "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run",
     "phpcs": "phpcs",
-    "pest": "vendor/bin/pest"
+    "pest": "pest"
```
Comment (szepeviktor) [inline composer.json:16]: Composer finds script in `bin-dir`.

File: composer.json [hunk 3]
```diff
@@ -24,6 +24,7 @@
     }
   },
   "require": {
+    "php": "^7.4 || ^8.0",
```
Comment (szepeviktor) [inline composer.json:27]: This is from the readme.

**Comment by seothemes** [comment]
Ha, awesome, thank you for this. I'll get them all merged today.

**Pull request merged**